cmd/crowdsec-cli/machines.go

```diff
@@ -151,19 +168,29 @@ The watcher will be validated automatically.
 			}
 		},
 		Run: func(cmd *cobra.Command, arg []string) {
+			var err error
+
+			// create machineID if doesn't specified by user
 			if machineID == "" {
-				log.Fatalf("please provide a machine id with --machine|-m ")
```

Comment:
I think we should require a flag for doing this, it might be tempting for the user to juste type `cscli machines add` to see options, and cobra usually will give help in this case. Can we require a flag such as `-a / --auto` or something for doing magic on the user's behalf